pandas/_libs/tslibs/period.pyx

```diff
@@ -21,26 +21,23 @@ from cpython.datetime cimport PyDateTime_Check, PyDateTime_IMPORT
 # import datetime C API
 PyDateTime_IMPORT
 
-from tslibs.np_datetime cimport (pandas_datetimestruct,
-                                 dtstruct_to_dt64, dt64_to_dtstruct,
-                                 is_leapyear)
-
+from np_datetime cimport (pandas_datetimestruct, dtstruct_to_dt64,
+                          dt64_to_dtstruct, is_leapyear)
 
 cimport util
 from util cimport is_period_object, is_string_object, INT32_MIN
 
-from missing cimport is_null_datetimelike
+from pandas._libs.missing cimport is_null_datetimelike
 from pandas._libs.tslib import Timestamp
```

Comment:
change this to 

```from timestamps import Timestamp``

Reply:
Good spot, I shouldn't have missed this, thanks!